scheduler/kubernetes/kafka-k8s.md

```diff
@@ -0,0 +1,873 @@
+---
+layout: page
+title: "Deploy Apache Kafka and Zookeeper with Portworx on Kubernetes"
+keywords: portworx, container, Kubernetes, storage, Docker, k8s, pv, persistent disk, kafka, zookeeper
+sidebar: home_sidebar
+---
+
+* TOC
+{:toc}
+
+These below instructions will provide you a step by step guide in deploying Apache Kafka and Zookeeper with Portworx on Kubernetes. 
+
+Kubernetes provides management of stateful workloads using Statefulsets. Apache Kafka is a distributed streaming platform. [Apache kafka](https://kafka.apache.org/) is used in multiple cases where the need might be to provide realtime streaming of data to systems or in using it as an enterprise messaging system. 
+Apache Kafka uses [Zookeeper](https://zookeeper.apache.org/) for maintaining configurations and distributed synchronization. 
+
+## Prerequisites
+
+-	A running Kubernetes cluster with v 1.6+ 
+-	All the kubernetes nodes should allow [shared volume propagation](https://docs.portworx.com/knowledgebase/shared-mount-propogation.html). PX requires this since it provisions volumes in containers.  
+-	[Deploy Portworx on your kubernetes cluster](https://docs.portworx.com/scheduler/kubernetes/install.html). PX runs on each node of your kubernetes cluster as a daemonset. 
+
+## Install
+
+### Portworx StorageClass for Volume Provisioning
+
+Portworx provides volume(s) to Zookeeper as well as Kafka. 
+Create ```portworx-sc.yml``` with Portworx as the provisioner and apply the configuration
+
+```
```

Comment:
Use "yaml" keyword after the 3 backticks. Do the same for all yaml docs in the document.